docs/marketplace/tutorials/create-a-rag-application.md

```diff
@@ -4,89 +4,130 @@ title: Create a RAG Application
 sidebar_label: Create a RAG Application
 ---
 
-In this tutorial, we will learn how to create a fully functioning RAG application in a Mia-Platform project in just a few minutes. We will cover the setup of the necessary services and the operations required to have a chatbot that is ready to assist and provide help on specific topics based on the documents provided.
+In this tutorial, you will learn how to create a fully functioning RAG application in a Mia-Platform project in just a few minutes.
+We will cover the setup of the necessary services and the operations required to have a chatbot that is ready to assist and provide help on specific topics based on the documents provided.
 
-To accomplish this, we will utilize the Marketplace application called **AI RAG Chat**. This application includes:
+To accomplish this, you will utilize the Marketplace application called **AI RAG Chat**. This application includes:
 
-- the **AI RAG Template** service, which leverages the APIs provided by [OpenAI](https://openai.com/) to receive answers to questions in natural language, and to create _embeddings_, encoded definitions coming from pieces of text extracted from a website.
+- the **RAG Chatbot API** service, which leverages the APIs provided by [OpenAI](https://openai.com/) or [Azure OpenAI](https://aka.ms/azure-openai) to receive answers to questions in natural language, and to create _embeddings_, encoded definitions coming from pieces of text extracted from a website.
 - the **AI RAG Template Chat** service, a small frontend application written in React that provides a simple chat interface to communicate with the chatbot.
 - the **API Gateway** service, that will be used to expose the endpoints to reach the frontend application and communicate with the service.
+- several endpoints to expose the necessary APIs from the services to use the application.
 
-Furthermore, we will install the **API Documentation Aggregator** application and the **API Portal** from the templates. These will enable us to easily send HTTP requests to the AI RAG Template service to store additional documents in a database in a way that can be used by the service to answer questions.
+:::info
+The _RAG Chatbot API_ is a plugin maintained and supported by Mia-Platform that includes all the necessary logic to build a fully functional RAG application.
+
+However, if you need to deep-dive into the code by yourself to create a more suitable and customized application, you can use the **AI RAG Chat Template** application.
+This application includes all the features above, but creates a clone of the [_AI RAG Template_](https://github.com/mia-platform/ai-rag-template) repository in your provider to allow you to customize it accordingly to your needs.
```

Comment:
```suggestion
The **AI RAG Chat Template** application includes all the features above, but creates a clone of the [_AI RAG Template_](https://github.com/mia-platform/ai-rag-template) repository in your provider to allow you to customize it accordingly to your needs.
```

Reply:
I think it's better to repeat the name just to avoid potential misunderstandings between the two different applications